src/base.ts

```diff
@@ -71,22 +74,69 @@ export default abstract class extends Command {
 
   got = got.extend()
   spinner!: ora.Ora;
-  readGlobalConfig(): IGlobalLiaraConfig {
-    let content
-
-    try {
-      content = JSON.parse(fs.readFileSync(GLOBAL_CONF_PATH).toString('utf-8')) || {}
-    } catch {
-      content = {}
+  async readGlobalConfig(): Promise<IGlobalLiaraConfig> {
+    if (fs.existsSync(GLOBAL_CONF_PATH)) {
+      fs.removeSync(PREVIOUS_GLOBAL_CONF_PATH);
+      const content = fs.readJSONSync(GLOBAL_CONF_PATH, {throws: false}) || {}
+      return content;
     }
-
-    // For backward compatibility with < 1.0.0 versions
-    if (content.api_token) {
-      content['api-token'] = content.api_token
-      delete content.api_token
+    const content = fs.readJSONSync(PREVIOUS_GLOBAL_CONF_PATH, {throws: false}) || {}
+    if (content.accounts && Object.keys(content.accounts).length) {
+      const accounts: IAccounts = {};
+      for (const account of Object.keys(content.accounts)) {
+        await this.setAxiosConfig({
+          "api-token": content.accounts[account].api_token,
+          region: content.accounts[account].region,
+        });
+        try {
+          const {
+            user: { email, fullname, avatar },
+          } = await got.get("v1/me").json<{ user: IAccount }>();
```

Comment:
I have a `.liara.json` file with valid tokens inside. I had some issues. Added a log to the catch section and I've got this error:

```
TypeError: Invalid URL
    at new NodeError (node:internal/errors:371:5)
    at onParseError (node:internal/url:552:9)
    at new URL (node:internal/url:628:5)
    at Object.exports.default (/home/mhe/projects/liara/cli/node_modules/got/dist/source/core/utils/options-to-url.js:35:17)
    at normalizeArguments (/home/mhe/projects/liara/cli/node_modules/got/dist/source/core/index.js:480:51)
    at got (/home/mhe/projects/liara/cli/node_modules/got/dist/source/create.js:112:39)
    at Function.got.<computed> [as get] (/home/mhe/projects/liara/cli/node_modules/got/dist/source/create.js:224:42)
    at AccountList.readGlobalConfig (/home/mhe/projects/liara/cli/src/base.ts:96:25)
    at AccountList.run (/home/mhe/projects/liara/cli/src/commands/account/list.ts:16:24)
    at AccountList._run (/home/mhe/projects/liara/cli/node_modules/@oclif/core/lib/command.js:52:22) {
  input: 'v1/me',
  code: 'ERR_INVALID_URL'
}
```

So I tried doing this:
```suggestion
          } = await this.get("v1/me").json<{ user: IAccount }>();
```

And it just fixed my issue.

Please fix this and add a corresponding test to verify it.